src/controller/securemgr/verifier/verifier.go

```diff
@@ -141,10 +144,9 @@ func (VerifierImpl) ContainerIsInWhiteList(containerName string) error {
 	if containerHashIsInWhiteList(containerName[index:]) {
 		log.Printf("%s container's hash: %s is in container white list\n", logPrefix, containerName[index:])
 		return nil
```

Comment:
In case of error, we return new error object, whereas in case of validated container we are returning nil. Some other logical return would be better i guess.

Reply:
Thanks for the comment! To return a different value, we need to add another return parameter. Could you give an example that you would like to see?

Reply:
As @suresh-lc's comment, it would be good to change some code. 
How about modifying as below?

1. func [containerHashIsInWhiteList](https://github.com/lf-edge/edge-home-orchestration-go/blob/beba02276bd9a563842b6b17d82c1ee84be29615/src/controller/securemgr/verifier/verifier.go#L104) :  return type from `bool`  to `error`
2. Move error logic here
```suggestion
		return errors.New("container's hash is not in container white list")
```
3. Move `return nil` to the last

Reply:
Got the point as suggested by @t25kim , but still we would return the nil. Since the function signature is to return error, guess returning nil would only suit it. Else we need to change the function signature might be something like

func vals() (int, int) {
    return 3, 7
}

and use whatever is needed like
 _, c := vals()

Reply:
As always, the Truth is in the middle.
1) I agree with the @t25kim and this simplifies the code at the moment
2) but for the future, I planned that the description of containers will contain additional information fields and here the @suresh-lc opinion is also correct.

Summary:
1) the code is working correctly now
2) I took note of your opinions as they are correct. Thank you!
3) In the near future I will think about whether we will limit more the container's policy. And we will discuss it again